### ISSUE TITLE
[Feature Request] Add Picture support

### ISSUE THREAD

**Issue by lo-mityaesh** [isjerryxiao/gnome-shell-extension-activate-gnome#15]
Hello, congratulations on your work, can you add the possibility of adding an image in addition to or instead of the text.
Thanks alot

**Comment by isjerryxiao** [comment]
I don't think this is what the extension should do, but if you think so, contributing is always possible.
